infra/development/docker/monitoring-docker-compose.yml

```
@@ -0,0 +1,36 @@
+      - 'grafana_storage:/var/lib/grafana'
+    depends_on:
+      - prometheus
+    networks:
+      - monitoring-network
+
+networks:
+  monitoring-network:
+    driver: bridge
+
+volumes:
+  grafana_storage: {}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)